kyo-core/shared/src/main/scala/kyo/StreamCoreExtensions.scala

```diff
@@ -82,8 +82,7 @@ object StreamCoreExtensions:
 
         def consume[S](stream: Stream[A, Abort[E] & S & Async])(
             using
-            i1: Isolate.Contextual[S, Sync],
-            i2: Isolate.Stateful[S, Abort[E] & Async],
+            i: Isolate[S, Sync, S],
```

Comment:
This change was motivated by the composability issues of the two isolates here + inference issue reported in https://github.com/getkyo/kyo/issues/1320. It's much more flexible now!

Reply:
💯, I think it's `Isolate[S, Async, S]` (not sure)